src/main/java/game/Game.java

```diff
@@ -0,0 +1,104 @@
+package game;
+
+import java.util.*;
+
+public class Game {
+    private Map<String, String> questions;
+    private String nextQuestion;
+    private List<String> currentOptions;
+    private boolean defeated;
+    private boolean victory;
+
+    public boolean isVictory() {
+        return victory;
+    }
+
+    public boolean isDefeated() {
+        return defeated;
+    }
+
+    public Game() {
+        initializeQuestions();
+        startGame();
+    }
+
+    public void initializeQuestions() {
+        questions = new HashMap<>();
```

Comment:
Path path = Path.of("resources/game.json");
var lines = Files.readAllLines(path);

ObjectMapper om = new ObjectMapper();
Root root = om.readValue(myJsonString, Root.class);